server/templates/static/homepage.html

```diff
@@ -26,258 +26,314 @@
+        averages. At every one of those levels, there will be places that become
+        much hotter and places that become colder.
+      </p>
+      <p>
+        Explore what temperatures might be:
+        <ul>
+          <li>
+            In the US in 2050 (<a
+            href="/tools/map#dt=2050-06&sv=DifferenceRelativeToBaseDate2006_Max_Temperature_RCP45&pc=0&denom=Count_Person&pd=country/USA&pt=Country&ept=County">optimistic</a>,
+            <a
+            href="/tools/map#dt=2050-06&sv=DifferenceRelativeToBaseDate2006_Max_Temperature_RCP85&pc=0&denom=Count_Person&pd=country/USA&pt=Country&ept=County">pessimistic</a>)<a href="#fn-rcp"><sup>1</sup></a>
+          </li>
+          <li>
+            In Europe might be in 2050 (<a
+            href="/tools/map#&sv=DifferenceRelativeToBaseDate2006_Max_Temperature_RCP45&pc=0&denom=Count_Person&pd=europe&pt=Continent&ept=IPCCPlace_50">optimistic</a>,
+            <a
+            href="/tools/map#&sv=DifferenceRelativeToBaseDate2006_Max_Temperature_RCP85&pc=0&denom=Count_Person&pd=europe&pt=Continent&ept=IPCCPlace_50">pessimistic</a>)<a href="#fn-rcp"><sup>1</sup></a>
+          </li>
+          <li>
+            Or the progression in Europe over time (<a
+            href="/tools/map#&sv=DifferenceRelativeToBaseDate2006_Max_Temperature_RCP45&pc=0&denom=Count_Person&dt=2040-06&pd=europe&pt=Continent&ept=IPCCPlace_50">2040</a>,
+            <a
+            href="/tools/map#&sv=DifferenceRelativeToBaseDate2006_Max_Temperature_RCP45&pc=0&denom=Count_Person&dt=2045-06&pd=europe&pt=Continent&ept=IPCCPlace_50">2045</a>,
+            <a
+            href="/tools/map#&sv=DifferenceRelativeToBaseDate2006_Max_Temperature_RCP45&pc=0&denom=Count_Person&dt=2050-06&pd=europe&pt=Continent&ept=IPCCPlace_50">2050</a>)
+          </li>
+        </ul>
+        <span class="clearfix"></span>
+      </p>
+      <p class="clearfix">
+        <span class="img-card float-md-left" style="width: 312px">
+          <a href="/tools/scatter#&svx=DifferenceRelativeToBaseDate2006_Max_Temperature_RCP45&svy=Percent_Person_WithCoronaryHeartDisease&epd=country/USA&ept=County&dd=1">
+            <img class="img-thumbnail" src="/images/home-us_heart_disease_temp.png" />
+          </a>
+          <small class="caption">Heart condition vs. max projected summer temperature for US counties (RCP 4.5) (source: CDC, NASA)</small>
+        </span>
+        Climate change is not just about reducing carbon emissions. It is also
+        about adaptation to the change that is already happening. Note the
+        outliers in the upper right quadrant of this chart: counties like <a
+        href="{{ url_for('place.place', place_dcid='geoId/46121') }}">Todd County,
+        SD</a> and <a href="{{ url_for('place.place', place_dcid='geoId/46102')
+        }}">Oglala Lakota County, SD</a> with high incidence of coronary disease can
+        also expect some of the highest the highest temperature rises.
```

Comment:
repeated "the highest"

Reply:
I wonder how to make the point more prominent? It's not obvious how the high temperature would affect coronary diesease?